catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -512,3 +527,10 @@ def _commit_records(self) -> int:
             total += store.commit()
         logger.info(f"Committed {total} records")
         return total
+
+    def _verbose_log(self, msg: str, data=[]):
```

Comment:
`list` in Python is mutable, so [it's tricky to use them as default for params](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). We want to avoid confusing situations so it's better to make the default here `None` (or remove the default).
```suggestion
    def _verbose_log(self, msg: str, data: list = None):
```